packages/components/src/modal/style.scss

```diff
@@ -66,13 +66,20 @@
 	justify-content: space-between;
 	background: $white;
 	align-items: center;
-	box-sizing: border-box;
```

Comment:
Was this just not needed anymore? It doesn’t seem related to the other changes.

Reply:
Yes, good catch, I should've noted that as a code comment. Turns out that rule was written out twice in the same rule. Here it is on line 65, still there: https://github.com/WordPress/gutenberg/blob/82276e622e458ec101e6364c214ee56588f5d865/packages/components/src/modal/style.scss#L61

Thank you for the review.